searchv2/builders_v3.py

```diff
@@ -0,0 +1,159 @@
+import json
+
+from datetime import timedelta
+from django.utils import timezone
+
+from grid.models import Grid
+from package.models import Commit, Package
+from searchv2.models import SearchV2
+from searchv2.rules import calc_package_weight
+from searchv2.rules import DeprecatedRule
+from searchv2.rules import DescriptionRule
+from searchv2.rules import DownloadsRule
+from searchv2.rules import ForkRule
+from searchv2.rules import LastUpdatedRule
+from searchv2.rules import RecentReleaseRule
+from searchv2.rules import UsageCountRule
+from searchv2.rules import WatchersRule
+from searchv2.utils import clean_title
+from searchv2.utils import remove_prefix
+
+
+def build_1(*, verbose: bool = False):
+    last_week = timezone.now() - timedelta(7)
+
+    SearchV2.objects.filter(created__lte=last_week).delete()
+
+    index_packages(verbose=verbose)
+    index_groups(verbose=verbose)
+
+    return SearchV2.objects.all()
+
+
+def index_packages(*, verbose: bool = False):
+    rules = [
+        DeprecatedRule(),
+        DescriptionRule(),
+        DownloadsRule(),
+        ForkRule(),
+        LastUpdatedRule(),
+        RecentReleaseRule(),
+        UsageCountRule(),
+        WatchersRule(),
+    ]
+
+    # demo the group rule
+
+    # group = ScoreRuleGroup(
+    #     name="Activity Rules",
+    #     description="Rules related to the package's recent activity",
+    #     max_score=40,
+    #     documentation_url="https://docs.yoursite.com/rules/groups/activity",
+    #     rules=[LastUpdatedRule(), RecentReleaseRule()],
+    # )
+
+    # rules = [
+    #     DeprecatedRule(),
+    #     DescriptionRule(),
+    #     DownloadsRule(),
+    #     ForkRule(),
+    #     UsageCountRule(),
+    #     WatchersRule(),
+    #     group,
+    # ]
```

Comment:
I would normally not merge in code like this, but we do want to switch over to the `ScoreRuleGroup` once we know everything is stable. This could be moved into an Issue too.